src/backend/distributed/commands/multi_copy.c

```diff
@@ -845,7 +1055,8 @@ OpenCopyTransactions(CopyStmt *copyStatement, ShardConnections *shardConnections
 	/* otherwise, mark failed placements as inactive: they're stale */
 	foreach(failedPlacementCell, failedPlacementList)
 	{
-		ShardPlacement *failedPlacement = (ShardPlacement *) lfirst(failedPlacementCell);
+		ShardPlacement *failedPlacement = (ShardPlacement *) lfirst(
+			failedPlacementCell);
 		uint64 shardLength = 0;
 
 		DeleteShardPlacementRow(failedPlacement->shardId, failedPlacement->nodeName,
```

Comment:
Ah, makes sense, comment should be fine, maybe even an Assert (!stopOnFailure || list_length(failedPlacementList) == 0);